code/stable/dblpend/SRS/PDF/DblPend_SRS.tex

```diff
@@ -23,6 +25,15 @@
 \newcommand{\lt}{\ensuremath <}
 \newlist{symbDescription}{description}{1}
 \setlist[symbDescription]{noitemsep, topsep=0pt, parsep=0pt, partopsep=0pt}
+\newsavebox{\mybox}
+\newcommand{\resizeExpression}[2]{
+\savebox{\mybox}{$#1$}
+\ifdim\wd\mybox>#2\linewidth
```

Comment:
can you make this format nicely, i.e. indent the stuff in the 'then' and 'else' branches?

Reply:
Fixed in https://github.com/JacquesCarette/Drasil/pull/3910/commits/3f29a2c9a1fed602ce065c77e3afc4b78af7ed87.